ecosystem/sep-0031.md

```diff
@@ -343,9 +426,12 @@ Name | Type | Description
 `id` | string | The ID returned from the `POST /transactions` request that created this transaction record.
 `status` | string | The status of the transaction. Values are outlined below.
 `status_eta` | number | (optional) The estimated number of seconds until a status change is expected.
-`amount_in` | string | (optional) The amount received or to be received by the Receiving Anchor. Up to 7 decimals are permitted. Excludes any fees charged after Receiving Anchor receives the funds.
-`amount_out` | string | (optional) The amount sent by the Rending Anchor to Receiving Client. Up to 7 decimals are permitted.
+`amount_in` | string | (optional) The amount of the Stellar asset received or to be received by the Receiving Anchor. Excludes any fees charged after Receiving Anchor receives the funds.
+`amount_in_asset` | string | (optional) The asset received or to be received by the Receiving Anchor. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`amount_out` | string | (optional) The amount sent by the Receiving Anchor to Receiving Client.
+`amount_out_asset` | string | (optional) The asset delivered to the Receiving Client. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
 `amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor.
+`amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
```

Comment:
I believe the idea now is all conversion fees are included in the quote returned from SEP-38 /price and /quote endpoints, right? In that case, we don't need the attribute `amount_fee_asset` here.

Reply:
I think it's worth keeping as an optional attribute in cash a fixed fee is also charged on top of the margin included in the exchange rate. If only the margin on the exchange rate is used, then `amount_fee` will be 0 anyway.